docs/source/terms_and_model.rst

```diff
@@ -342,11 +342,122 @@ Location for Variation.
 **Implementation Guidance**
 
 * Location refers to a position.  Although it MAY imply a sequence,
-  the two concepts are not interchangable, especially when the
+  the two concepts are not interchangeable, especially when the
   location is non-specific (e.g., a range) or symbolic (a gene).
 
 
+.. _chromosome-location:
+
+ChromosomeLocation
+$$$$$$$$$$$$$$$$$$
+
+**Biological definition**
+
+Imprecise chromosomal locations based on named landmarks assayed
+through chromosomal staining, FISH, recombination studies, or other
+low-resolution methods.
+
+**Computational definition**
+
+A ChromosomeLocation is a :ref:`Location` that is defined by a named
+chromosomal band within a species.
+
+**Information model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Location Id; MUST be unique within document
+   * - type
+     - string
+     - 1..1
+     - Location type; MUST be set to **'ChromosomeLocation'**
+   * - species
+     - :ref:`CURIE`
+     - 1..1
+     - An external reference to a species taxonomy.  See Implementation Guidance, below.
+   * - chr
+     - string
+     - 1..1
+     - The symbolic chromosome name
+   * - start
+     - string
+     - 1..1
+     - The start chromosomal band name
+   * - end
+     - string
+     - 1..1
+     - The end chromosomal band name
+
+
+**Implementation guidance**
+
+* ChromosomeLocation is intended to enable the representation of
+  cytogenetic results from karyotyping or low-resolution molecular
+  methods.  Precise :ref:`SequenceLocation` should be preferred when
+  nucleotide-scale location is known.
+* `species` is specified using the NCBI taxonomy.  The CURIE prefix
+  MUST be `taxonomy`, corresponding to the `NCBI taxonomy prefix at
```

Comment:
Note that the proposed structure of the CURIE (`taxonomy:<identifier>`) differs from NCBI's suggested syntax for references within manuscripts.  Example for [humans](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=9606&lvl=3&lin=f&keep=1&srchmode=1&unlock):

> Taxonomy ID: 9606 (for references in articles please use NCBI:txid9606)

In that form, "NCBI" is the prefix and the reference is a concatenation of "txid" and the taxonomy identifier.

I do not think VRS needs to follow the NCBI recommendations for xrefs in manuscripts, but it is something we should be aware of (and possibly mention within the docs as a known divergence).

Reply:
Ultimately, CURIEs are local decisions that we hope will follow convention. We currently standardize on prefixes registered in identifiers.org, and I think that's preferable to deferring to each data source.